explainers/subresource-loading.md

```diff
@@ -166,51 +160,34 @@ is a detailed exploration of how to efficiently retrieve only updated resources
 on the second load. The key property is that the client's request for a bundle
 embeds something like a [cache
 digest](https://httpwg.org/http-extensions/cache-digest.html) of the resources
-it already has within the bundle's scope, and the server sends down the subset
-of the bundle that the client doesn't already have.
+it already has, and the server sends down the subset of the bundle that the
+client doesn't already have.
 
-## Authority
+## Compressed list of resources
 
-As with [navigation to unsigned
-bundles](https://github.com/WICG/webpackage/blob/master/explainers/navigation-to-unsigned-bundles.md#loading-an-authoritative-unsigned-bundle),
-a bundle's responses aren't always
-[authoritative](https://httpwg.org/http-core/draft-ietf-httpbis-semantics-latest.html#establishing.authority)
-for their claimed URLs. Responses on the bundle's origin and (by default) under
-the bundle's path are authoritative. Responses with a claimed URL that is a hash
-of the response itself, like the [`ni:///`
-scheme](https://tools.ietf.org/html/rfc6920), could also be authoritative.
-
-Unlike in the navigation case, but like with Service Workers, it may be
-acceptable to let a page lie to itself and use non-authoritative responses from
-bundles it has selected. However, we do not intend to allow this for now.
+As discussed in [Dynamic bundle serving with
+WebBundles](https://docs.google.com/document/d/11t4Ix2bvF1_ZCV9HKfafGfWu82zbOD7aUhZ_FyDAgmA/edit),
+simply including a list of resources in the HTML [may cost as little as 5 bytes
+per URL on average after the HTML is
+compressed](https://github.com/yoavweiss/url_compression_experiments).
 
 ## Alternate designs
 
-### Defining the scope
-
-The approach above of just defining a scope directory doesn't allow much
-flexiblity in how resources are grouped into bundles. Several other scoping
-mechanisms are available to give the bundler more flexibility, at the cost of
-being bigger and less readable.
+Several other mechanisms are available to give the bundler more flexibility.
```

Comment:
```suggestion
### Summarizing the contents of the bundle

Several other mechanisms are available to give the bundler more flexibility or to compress the resource list.
```

And then keep the subsections about this nested with `####`.

Reply:
Done! Thanks!